### ISSUE TITLE
Button Status

### ISSUE THREAD

**Issue by zcjess** [zcjess/Atlas#6]
Should announce the different states of these buttons, for example:
`<button disabled aria-label="Submit (currently disabled due to incomplete form)">Submit</button>`
![image](https://github.com/zcjess/Atlas/assets/90011962/a5088b7d-329a-4a2f-8054-6ee7b51a96a7)

<img width="73" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/270189f7-d680-4d35-ba12-8c6a6dca2e50">
https://atlas-dev.ai.umich.edu/course/AAS%20103/
<img width="94" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/477ab76f-c7f1-43bc-8d8d-897915bc9446">
https://atlas-dev.ai.umich.edu/my-dashboard/
<img width="163" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/e028df4e-9b3a-4f32-b6cf-bb49f110b028">
https://atlas-dev.ai.umich.edu/about/

announce states of bookmark
<img width="172" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/13be5f84-0d0c-42fa-829d-2f1061c2572e">
https://atlas-dev.ai.umich.edu/course/AAS%20103/

**Comment by zcjess** [comment]
Should have aria-expanded and aria-haspopup role
The school filter can also use the search box as in the subject filter
![image](https://github.com/zcjess/Atlas/assets/90011962/2d950885-87e6-4dfa-8599-fb97a06bab7d)

popup announcement
<img width="289" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/d074b3a8-168a-40b0-8129-4df6c34aaacf">

Have no way to close popup
![image](https://github.com/zcjess/Atlas/assets/90011962/e8498987-2baf-43c2-a8da-2cbe433b88b8)

**Comment by zcjess** [comment]
Announce the result after clicking "add" (new collection xxx is added and checked)
<img width="341" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/44aecb75-f420-4477-b382-39ce85d6cc02">

**Comment by zcjess** [comment]
Announce the result of clicking this button
<img width="796" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/d7889569-a112-49fd-879d-714cc9d7c3df">

**Comment by zcjess** [comment]
Better to set one button to be disabled if they are not applicable at certain states
<img width="305" alt="image" src="https://github.com/zcjess/Atlas/assets/90011962/9f12bea9-06d1-47d2-8e25-439039b52274">
